### ISSUE TITLE
does it support append file?

### ISSUE THREAD

**Issue by sahas-** [jprichardson/node-jsonfile#52]
is there a way to do append new objects to a file?

**Comment by ajsingh007** [comment]
I have the same question. Its working well for me but it overwrites the existing json rather than adding to it

**Comment by dudleyjosh** [comment]
Same here... this would be a perfect little json to file utility if it included fs.appendFile() and fs.appendFileSync() functionality too.

**Comment by SCBuergel** [comment]
:+1: without this I cant use it.

**Comment by kamikazechaser** [comment]
please support append file

**Comment by andela-anandaa** [comment]
See [my comments here](https://github.com/jprichardson/node-jsonfile/issues/67#issuecomment-287752793), I think it's related.

I had a similar issue, wanted to to hack our `package.json` file to update the `browser` key depending on the environment, and this approach worked okay. 

**TL;DR - read the file in the object, update the update, and write the object back to the file afresh.**

**Comment by RyanZim** [comment]
As discussed in https://github.com/jprichardson/node-jsonfile/issues/67, you can use `Object.assign` to accomplish something like this. You can't truly append to a JSON file in the filesystem sense, so I don't see the point of adding this syntax sugar to jsonfile.

@jprichardson what are your thoughts here?

**Comment by jprichardson** [comment]
> @jprichardson what are your thoughts here?

Agreed. I'd prefer to rework (move to its own section) the README a bit to make it very clear and distinct on the two cases of appending or updating.
